lib/3scale/backend/csp.rb

```diff
@@ -0,0 +1,31 @@
+require '3scale/backend/headers/stringify'
+
+# CSP support
+#
+# Please see references:
+#
+#     https://content-security-policy.com/
+#     https://developer.mozilla.org/en-US/docs/Web/HTTP/CSP
+#     https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy
+#     https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/Sources
+module ThreeScale
+  module Backend
+    module CSP
+      extend Headers::Stringify
+
+      CSP_VALUES = "default-src 'self'".freeze
+      private_constant :CSP_VALUES
+
+      CSP_HEADERS = {
+        'Content-Security-Policy'.freeze => CSP_VALUES
+      }.freeze
+      private_constant :CSP_HEADERS
+
+      stringify_consts :CSP_VALUES, :CSP_HEADERS
```

Comment:
what is this stringify supposed to do?

Reply:
I don't know, I took it from https://github.com/3scale/apisonator/blob/4f29aabd1dbd71c9eb06403614080aa42feb3d01/lib/3scale/backend/cors.rb#L53

I assumed it was needed for something, but I could be wrong.

Reply:
I read it as if it creates a constant `CONST_NAME_S` with stringified value. I see no reason to do that because we use the original constant only as far as I can see.

Reply:
Yeah, no idea why is this here. Probably dead code